### PR TITLE
[test] Update simd checks for Alps

### PR DESCRIPTION
**Pull request by jgphpc**
Tested with nsimd/3b6e339-cpeGNU-20.10 and nsimd/3b6e339-CrayGNU-20.10

### CODE REVIEW

**Comment by jgphpc** [comment]
retest this

File: cscs-checks/microbenchmarks/cpu/simd/benches/benches.hpp [hunk 1]
```diff
@@ -0,0 +1,72 @@
+#ifndef BENCHES_HPP
```
Comment (ekouts) [inline cscs-checks/microbenchmarks/cpu/simd/benches/benches.hpp:1]: Is this file ours? Should it have the copyright header?

**Review by ekouts — CHANGES_REQUESTED** [review]

File: cscs-checks/microbenchmarks/cpu/simd/benches/benches.hpp (hunk 1 above)
Comment (jgphpc) [reply cscs-checks/microbenchmarks/cpu/simd/benches/benches.hpp:1]: it's from the [simd repo](https://github.com/agenium-scale/nsimd/blob/master/benches/benches.hpp)
What do you think is the best ?

Comment (vkarak) [reply cscs-checks/microbenchmarks/cpu/simd/benches/benches.hpp:1]: Perhaps we can add a single line comment pointing to the file on that repo.

File: cscs-checks/microbenchmarks/cpu/simd/nsimd.py [hunk 2]
```diff
@@ -7,98 +7,52 @@
...
-            'tar xf $EBROOTNSIMD/benches.tar benches/benches.hpp',
-            'tar xf $EBROOTNSIMD/benches.tar %s' % self.sourcepath,
-        ]
-        self.executable = '%s.exe' % self.testname
-        self.modules = ['nsimd/579084-CrayGNU-19.10']
+        self.testname = testname
+        # c++ test code generated with:
+        #   python3 egg/hatch.py --benches --simd avx2 -m mul
+        self.sourcesdir = 'benches'
+        self.sourcepath = self.testname
```
Comment (vkarak) [inline cscs-checks/microbenchmarks/cpu/simd/nsimd.py:44]: Just to be consistent with the line below.

```suggestion
        self.sourcepath = testname
```

**Review by vkarak — APPROVED** [review]
lgtm. I have just two minor comments.

**Comment by vkarak** [comment]
@jgphpc The modules are not present on Eiger, are they?

**Comment by jgphpc** [comment]
@vkarak not installed but the recipes are available: 
https://github.com/eth-cscs/production/blob/master/easybuild/easyconfigs/n/nsimd/nsimd-3b6e339-cpeGNU-20.10.eb

File: cscs-checks/microbenchmarks/cpu/simd/benches/benches.hpp (hunk 1 above)
Comment (jgphpc) [reply cscs-checks/microbenchmarks/cpu/simd/benches/benches.hpp:1]: added a comment in the check

**Review by vkarak — APPROVED** [review]
lgtm

**Comment by vkarak** [comment]
@ekouts Can you approve this one?

**Review by ekouts — APPROVED** [review]
lgtm

**Pull request merged**